tools/run_tests/performance/loadtest_config.py

```diff
@@ -378,6 +440,18 @@ def main() -> None:
                       '--output',
                       type=str,
                       help='Output file name. Output to stdout if not set.')
+    argp.add_argument('--client_channels',
+                      type=int,
+                      help='Number of client channels.')
+    argp.add_argument('--server_threads',
+                      type=int,
+                      help='Number of async server threads.')
+    argp.add_argument(
+        '--offered_loads',
+        nargs="*",
+        type=int,
+        default=[],
+        help='A list of offered loads that the tests are running with.')
```

Comment:
This is a bit vague. How about "A list of QPS values at which each load test scenario will be run."?